# FusariumBase
Omics analysis workflow and code involved in FusariumBase

FusariumBase: an integrated web-based multi-omics database and analysis tool.

To help researchers make systematic use of Fusarium spp. data resources, we have created a specialized database called FusariumBase (https://FusariumBase.com). 
This all-encompassing multi-omics platform consists of four key sections: 

1) Multi-omics data: The database brings together high-quality genomes from 335 species, complete with thorough structural and functional annotations. Moreover, we have produced structures for more than 40,000 secreted proteins and have documented the temporal expression profiles for wheat, maize, and soybean infected by the major agricultural pathogen Fusarium graminearum. The database also includes 1,291 multi-conditional transcriptome datasets from 34 Fusarium species, which allows for the creation of gene co-expression networks to conduct detailed functional studies, such as investigating the downstream targets of transcription factors (TFs).

2) Phenotypic data: We have gathered morphological data on 221 Fusarium species from the published literature, conducted standardized tests to evaluate the pathogenicity of 102 Fusarium species on wheat, maize, and soybean, and compiled distribution and habitat details for 43,321 samples from around the world.

3) User-friendly toolkit: The database offers seven standard analysis tools, such as a gene function search, gene expression profile search, gene cluster visualization, GO/KEGG enrichment analysis, PCR primer design, a sequence alignment tool (BLAST), and pan-genome visualization. 

4) Analysis pipelines: Furthermore, we have developed two custom analysis pipelines. The first one uses various machine learning models to forecast the pathogenicity of Fusarium species without the need for actual strains, which is vital for quickly identifying and evaluating biosecurity risks in diverse settings. The second one effectively and precisely identifies Fusarium species by constructing phylogenetic trees based on multiple consecutive conserved sequences.
